gax-java/gax/src/main/resources/META-INF/native-image/com.google.api/gax/native-image.properties

```diff
@@ -1,6 +1,9 @@
 Args = --enable-url-protocols=https,http \
 --initialize-at-build-time=org.conscrypt,\
-  org.junit.platform.engine.TestTag \
+  org.junit.platform.engine.TestTag,\
+  com.google.api.gax.core.GaxProperties,\
+  com.google.common.base.Platform,\
+  com.google.common.base.Platform$JdkPatternCompiler \
```

Comment:
Were these transitively initialized at build time due to GaxProperties?

Reply:
yes via the Strings function